lib/vanagon/platform/windows.rb

```diff
@@ -34,6 +34,143 @@ def package_name(project)
         "#{project.name}-#{project.version}-#{project.release}-#{@architecture}.msi"
       end
 
+      def add_repository(definition)
+        definition = URI.parse(definition)
```

Comment:
(And if it's not always going to be a URL, just passing it to `URI` might explode hilariously)